src/objects.js

```diff
@@ -1,41 +1,85 @@
 const createPerson = (name, age) => {
   // your code here
+  return person = {
+    name:name,
+    age:age,
+  };
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  if (object.hasOwnProperty(property) == true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 const isOver65 = person => {
   // your code here
+  if (person.age > 65) {
+    return true; 
+  } else {
+    return false;
+  }
 };
 
 const getAges = people => {
   // your code here
+  let arrAges = []
+  for (let i = 0; i < people.length; i++) {
+      arrAges.push(people[i].age);
+    };
+  return arrAges;
 };
 
 const findByName = (name, people) => {
   // your code here
+  for (let i = 0; i < people.length; i++) {
+    if (people[i].name == name) {
```

Comment:
In this approach use === instead of ==.


It could also be done with the find method:
`````
return people.find(person => person.name === name)
``````